ihatemoney/tests/budget_test.py

```diff
@@ -917,6 +917,11 @@ def test_statistics(self):
         # Add a participant with a balance at 0 :
         self.client.post("/raclette/members/add", data={"name": "pépé"})
 
+        # Check that the "monthly expenses" table is empty
+        response = self.client.get("/raclette/statistics")
+        regex = r"<thead>\s*<tr>\s*<th>Period</th>\s*<th>Spent</th>\s*</tr>\s*</thead>\s*<tbody>\s*</tbody>"
```

Comment:
I'm probably rusted for not reading regexp for some time, but are we sure that a non-empty table will not pass the regex?

Reply:
Definitely, yes, the \s* only matches whitespace and newlines. So it will only match if there is an empty body.